ui/src/theme.ts

```diff
@@ -0,0 +1,84 @@
+import { createTheme } from '@mui/material/styles';
+
+export const PRIMARY_MAIN = '#4B50B4';
+export const PRIMARY_LIGHT = '#C9CDFF';
+export const PRIMARY_LIGHTER = '#DDE0FF';
```

Comment:
Can we not export these colors? Thinking it might create confusion in the future. Like onee file import directly from this, and some other files import from the customTheme object.

Not sure if erasing the export will make it error, pls help check :)

Reply:
okii